lldb/source/Plugins/Process/Windows/Common/ProcessWindows.cpp

```diff
@@ -203,11 +203,17 @@ ProcessWindows::DoAttachToProcessWithID(lldb::pid_t pid,
   return error;
 }
 
-Status ProcessWindows::DoResume() {
+Status ProcessWindows::DoResume(RunDirection direction) {
```

Comment:
We should use `DoResumeReverse()` in Process.h and this change won't need to happen.

Reply:
The decision was to use a direction flag.